src/components/periods/Periods.vue

```diff
@@ -171,6 +171,9 @@ export default {
 				this.loadingSave = false;
 				this.closeDialog();
 			}
+			//agrego esta linea ya que apretar el botton guardar sin ingresar nada en los campos
+			//el loadingSave queda true y no se sale.
+			this.loadingSave = false;
```

Comment:
Buen detalle, fue un error que se me escapo. podras mejor bajar `this.loadingSave = true;` dentro del if es decir si pasa la validacion a la primera linea. y ya no sera necesario ese this.loadingSave = false;

Reply:
listo.